docs/source/guides/development/getting_started.rst

```diff
@@ -61,32 +61,37 @@ contracts (https://docs.nucypher.com/en/latest/architecture/contracts.html).
 For general background information about choosing a node technology and operation,
 see https://web3py.readthedocs.io/en/stable/node.html. 
 
-In this guide, a local Geth node connected to the Goerli Testnet is used.
+In this guide, a local Geth node connected to Ethereum mainnet.
 For detailed information on using the geth CLI and Javascript console,
 see https://geth.ethereum.org/interface/Command-Line-Options.
 
-To run a Goerli-connected Geth node in *fast* syncing mode:
+.. important::
+
+    While the example provided uses Ethereum mainnet, these steps can be followed for the Goerli Testnet
+    with updated `geth` (``~/.ethereum/goerli/geth.ipc``) and `seed` uris
```

Comment:
Same q as above - we give the updated geth but not the seed uri.

Reply:
✔️